README.md

```diff
@@ -1,21 +1,23 @@
+# Ninja
+
 Ninja is a small build system with a focus on speed.
 https://ninja-build.org/
 
-See the manual -- https://ninja-build.org/manual.html or
-doc/manual.asciidoc included in the distribution -- for background
+See [the manual](https://ninja-build.org/manual.html) or
+`doc/manual.asciidoc` included in the distribution for background
```

Comment:
included in the official Ninja release artifacts for background

Reply:
It's actually not in the release artifacts -- they only contain the binary.

Reply:
Then what is "the distribution"?

Reply:
Source tarball or Git clone.

Reply:
Would it make sense to say "source tarball or git clone" instead of "the distribution" ?

Reply:
Dunno. I don't really want to bike-shed over the README in this PR though.

Reply:
*shrug*. If none of the project maintainers know what the sentence means why include it?

Reply:
I'm not including it, just changing the formatting to Markdown.

Reply:
Er. The sentence is still in the result. That's all I meant.